src/main/java/thrift/model/util/CurrencyUtil.java

```diff
@@ -57,11 +95,32 @@ public static double convertToDefaultCurrency(Map<String, Double> currencyMappin
         requireNonNull(currencyMappings);
         requireNonNull(value);
         requireNonNull(currency);
-        if (currencyMappings.containsKey(currency)) {
-            return (value / currencyMappings.get(currency));
+        if (currencyMappings.containsKey(currency.toUpperCase())) {
+            return (value / currencyMappings.get(currency.toUpperCase()));
         } else {
             return value;
         }
     }
 
+    /**
+     * Converts input currency amount from {@code currencyFrom} to {@code currencyTo} via {@link Value#DEFAULT_CURRENCY}
+     * @param currencyMappings Mapping of currency rates scaling from
```

Comment:
It should have an empty line between description and parameter section.

Reply:
Resolved.